cfme/fixtures/v2v.py

```diff
@@ -20,15 +21,23 @@ def v2v_providers(request, second_provider, provider):
 @pytest.fixture(scope='function')
 def host_creds(request, v2v_providers):
     """Add credentials to conversation host"""
-    provider = v2v_providers[1]
-    hosts = provider.hosts.all()
     try:
+        vmware_provider = v2v_providers[0]
+        hosts = vmware_provider.hosts.all()
+        for host in hosts:
+            host_data, = [data for data in vmware_provider.data['hosts']
+             if data['name'] == host.name]
+            host.update_credentials_rest(credentials=host_data['credentials'])
+
+        rhv_provider = v2v_providers[1]
+        hosts = rhv_provider.hosts.all()
```

Comment:
It seems to me that you are overwriting any changes you made for the vmware host creds here?